Solutions/Corvus.Tenancy.Abstractions/Corvus/Tenancy/TenantExtensions.cs

```diff
@@ -123,7 +123,7 @@ public static string EncodeGuid(Guid guid)
         /// </summary>
         /// <param name="bytes">The bytes to encode.</param>
         /// <returns>A string composed of pairs of hex digits for the byte array.</returns>
-        public static string ByteArrayToHexViaLookup32(byte[] bytes)
+        public static string ByteArrayToHexViaLookup32(this byte[] bytes)
```

Comment:
I know we only get this if we `using` whatever namespace this is in, but even so I question whether this or the preceding method are suitable candidates for being an exension methods. Is this really such a common thing to want to do with a `byte[]` that I need it to be something that pops up automatically as a pseudo member? (I'm not even really sure what the point of these two transformations is.)

Reply:
I would tend to agree.